server/routerlicious/packages/routerlicious-base/src/riddler/api.ts

```diff
@@ -91,7 +91,13 @@ export function create(
 	router.get("/tenants/:id/keys", (request, response) => {
 		const tenantId = request.params.id;
 		const includeDisabledTenant = getIncludeDisabledFlag(request);
-		const tenantP = manager.getTenantKeys(tenantId, includeDisabledTenant);
+		const getPrivateKeys = getPrivateKeysFlag(request);
+		const tenantP = manager.getTenantKeys(
+			tenantId,
+			includeDisabledTenant,
+			false,
```

Comment:
nit: For readability, add the parameter name as a comment corresponding to the argument value "false."